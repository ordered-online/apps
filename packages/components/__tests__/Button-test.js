import React from 'react';
import renderer from 'react-test-renderer';

import { Button } from '../';

describe('App', () => {
  it('Button renders correctly', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
