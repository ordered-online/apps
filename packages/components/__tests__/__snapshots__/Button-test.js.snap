// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Button renders correctly 1`] = `
<View
  accessibilityRole="button"
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "marginVertical": 10,
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {},
        Object {
          "backgroundColor": "#57c75e",
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 18,
            "margin": 8,
            "textAlign": "center",
          },
        ]
      }
    />
  </View>
</View>
`;
